Fixes an atmos related runtime involving rotating machineries

vincentiusvin opened the pull request:
## About The Pull Request
`nullifyPipenet` is called without checking if `parents[1]` is actually filled. If you use wrench faster than SSair rebuilds pipenets you might get runtimes.

## Why It's Good For The Game
Less runtimes

## Changelog
:cl:
fix: fixed cryocells and hfr runtiming on quick consecutive wrench actions.
/:cl:

vincentiusvin committed:
Adds a simple if check to prevent runtimes

vincentiusvin committed:
Deletes a mistaken comment that I added in the prev commit

LemonInTheDark commented:
This a problem anywhere else?

vincentiusvin commented:
> This a problem anywhere else?

Doing a quick search on `nullifyPipenet`, it seems like most of the implementations of it already have an if check. Only HFR, TEG, and Cryo lacks one. Would you like them bundled in too?

LemonInTheDark commented:
Yes I would, thank you. At some point we should make a helper proc for this shit, but that is a problem for another day

vincentiusvin committed:
Implemented the check for TEG and HFR.
Also moved cryo nullifyPipenet check and firing to only happen if a node is present.

vincentiusvin commented:
I added the other two machines, and put the check inside `if(node)`. Some other implementations however have those outside. Which one do you think would be better? Should the temporary pipelines be qdeled by `nullifyPipenet` directly or handled by the subsystem?
